src/components/profile/profile.stories.mdx

```diff
@@ -51,6 +63,14 @@ To use an image, simply pass any valid image URL as a `src` prop.
   <Story name="src" story={stories.Src} />
 </Canvas>
 
+### Gravatar
```

Comment:
**question**: probably one for the wider team, but even though this has always worked and we definitely shouldn't remove it as part of this PR, I'm not sure explicitly documenting the `gravatar` functionality for the first time at this stage is a good idea. I'm pretty sure we have mentioned removing/deprecating this feature (from `Portrait`, which would obviously also stop it working here).

Reply:
Agree, i'll remove this story example

Reply:
Yeah I agree, I think we can look to deprecate this soon so lets not document it and inadvertently encourage it's use